2022-01-20 14:29:15,960 INFO    MainThread:254881 [wandb_setup.py:_flush():71] setting env: {}
2022-01-20 14:29:15,960 INFO    MainThread:254881 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 14:29:15,960 INFO    MainThread:254881 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 14:29:15,960 INFO    MainThread:254881 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220120_142915-1cegu4t8/logs/debug.log
2022-01-20 14:29:15,961 INFO    MainThread:254881 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220120_142915-1cegu4t8/logs/debug-internal.log
2022-01-20 14:29:15,961 INFO    MainThread:254881 [wandb_init.py:init():404] calling init triggers
2022-01-20 14:29:15,961 INFO    MainThread:254881 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-20 14:29:15,962 INFO    MainThread:254881 [wandb_init.py:init():460] starting backend
2022-01-20 14:29:15,962 INFO    MainThread:254881 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-20 14:29:15,976 INFO    MainThread:254881 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 14:29:15,981 INFO    MainThread:254881 [backend.py:ensure_launched():221] started backend process with pid: 254936
2022-01-20 14:29:15,981 INFO    MainThread:254881 [wandb_init.py:init():469] backend started and connected
2022-01-20 14:29:15,983 INFO    MainThread:254881 [wandb_init.py:init():533] updated telemetry
2022-01-20 14:29:15,991 INFO    MainThread:254881 [wandb_init.py:init():563] communicating current version
2022-01-20 14:29:16,328 INFO    MainThread:254881 [wandb_init.py:init():568] got version response 
2022-01-20 14:29:16,328 INFO    MainThread:254881 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 14:29:16,515 INFO    MainThread:254881 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 14:29:16,596 INFO    MainThread:254881 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 14:29:16,596 INFO    MainThread:254881 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-20 14:29:16,596 INFO    MainThread:254881 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-20 14:29:16,597 INFO    MainThread:254881 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-20 14:29:16,597 INFO    MainThread:254881 [wandb_init.py:init():633] run started, returning control to user process
2022-01-20 14:29:20,586 INFO    MainThread:254881 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-20 14:29:20,587 INFO    MainThread:254881 [wandb_run.py:_restore():1752] restore
2022-01-20 14:29:22,726 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 953
  total_bytes: 953
}

2022-01-20 14:29:22,955 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 953
  total_bytes: 953
}

2022-01-20 14:29:23,525 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 953
  total_bytes: 8550
}

2022-01-20 14:29:23,627 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 953
  total_bytes: 8550
}

2022-01-20 14:29:23,729 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2634
  total_bytes: 8550
}

2022-01-20 14:29:23,833 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2634
  total_bytes: 8550
}

2022-01-20 14:29:23,936 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8550
  total_bytes: 8550
}

2022-01-20 14:29:24,039 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8550
  total_bytes: 8550
}

2022-01-20 14:29:24,142 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8550
  total_bytes: 8550
}

2022-01-20 14:29:24,246 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8550
  total_bytes: 8550
}

2022-01-20 14:29:24,991 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8550
  total_bytes: 8550
}

2022-01-20 14:29:25,385 INFO    MainThread:254881 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8550
  total_bytes: 8550
}
local_info {
}

2022-01-20 14:29:26,436 INFO    MainThread:254881 [wandb_run.py:_append_files():2180] logging synced files
